gradle.properties

```diff
@@ -4,10 +4,8 @@ profile=Java
 # Branch codename (filled by releases)
 codename=bleeding_edge
 # Run findbugs
-runFindbugs=false
-# Produce java6 bytecode only if running on top of jenkins and JAVA_6_HOME is available
-doJava6Bytecode=auto
-# Run animal sniffer - by default false, but if java6 bytecode is requested 
+doFindbugs=false
+# Run animal sniffer - by default false, but if java6 bytecode is requested
```

Comment:
yes, i would like to keep animal sniffer inside - it could be configured to check against Java7 API compatibility